src/storage/mvcc/reader/point_getter.rs

```diff
@@ -151,14 +155,24 @@ impl<S: Snapshot> PointGetter<S> {
             }
         }
 
+        let mut can_be_cached_by_lock = true;
```

Comment:
It's ugly, need help...

Reply:
Here it's also not performant... If we want to check cacheability, we seek twice...

Reply:
We have to seek twice, I think.
near_seek might help a little.